core/templates/dev/head/domain/topic_viewer/topic-viewer-backend-api.service.ts

```diff
@@ -20,20 +20,24 @@ require('domain/utilities/url-interpolation.service.ts');
 
 require('domain/topic_viewer/topic-viewer-domain.constants.ajs.ts');
 
+import { TopicDataObjectFactory } from
+  'domain/topic_viewer/TopicDataObjectFactory';
+
 angular.module('oppia').factory('TopicViewerBackendApiService', [
   '$http', '$q', 'UrlInterpolationService', 'TOPIC_DATA_URL_TEMPLATE',
   function($http, $q, UrlInterpolationService, TOPIC_DATA_URL_TEMPLATE) {
```

Comment:
The service should be imported in the angular module, right?

@bansalnitish, can you please confirm this?

Reply:
What's this? I haven't followed the complete PR but we are not using an updated service like this in place of AngularJS service.

We add downgrade statement in Angular service and then use it using require statements itself. Let me know if you have some questions.

Reply:
Oh its `require('domain/topic_viewer/TopicDataObjectFactory')` in service and `angular.module('oppia').factory(... , downgradeInjectable(...))` in Class file